Filters/exceptions.txt

```diff
@@ -2,6 +2,10 @@
 ! Once you enable private DNS, Android 9 starts resolving random domains looking like
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/204
+@@||cj.dotomi.com^|
+@@||emjcd.com^|
```

Comment:
Better use excluding instead of unblocking if it is possible
`||emjcd.com^`
`||dotomi.com^`

`||track.webgains.com^` is our rule. Better remove it, if it breaks(check in Safari)